src/Providers/FeatureLayer.js

```diff
@@ -95,7 +95,11 @@ export var FeatureLayerProvider = FeatureLayerService.extend({
       queryString.push(field + " LIKE upper('%" + text + "%')");
     }
 
-    return queryString.join(' OR ');
+    if (this.options.where) {
+      return this.options.where + ' AND ' + queryString.join(' OR ');
```

Comment:
thanks for holding my hand through that.  my brain wasn't firing on all cylinders yesterday so i only tested w/ two distinct words *within* the search.

resolved by #144.